pallets/pallet-web3-names/src/lib.rs

```diff
@@ -360,8 +358,8 @@ pub mod pallet {
 		pub fn update_deposit(origin: OriginFor<T>, name_input: Web3NameInput<T>) -> DispatchResult {
 			let source = ensure_signed(origin)?;
 			let name = Web3NameOf::<T>::try_from(name_input.into_inner()).map_err(DispatchError::from)?;
-			let w3n_entry = Owner::<T>::get(&name).ok_or(Error::<T>::Web3NameNotFound)?;
-			ensure!(w3n_entry.deposit.owner == source, Error::<T>::Unauthorized);
+			let w3n_entry = Owner::<T>::get(&name).ok_or(Error::<T>::NotFound)?;
+			ensure!(w3n_entry.deposit.owner == source, Error::<T>::NotAuthorized);
```

Comment:
There still exists _Unauthorized_ in the attestation pallet which hasn't been touched in terms of authorization errors. Please check all pallets again for similar occurrences.

Reply:
I checked, its all gone now